ci/build_cpp.sh

```diff
@@ -7,10 +7,15 @@ source rapids-env-update
 
 export CMAKE_GENERATOR=Ninja
 
+LIBCUDF_CHANNEL=$(rapids-get-artifact ci/cudf/pull-request/12587/046025a/cudf_conda_cpp_cuda11_$(arch).tar.gz)
+
 rapids-print-env
 
 rapids-logger "Begin cpp build"
 
-rapids-mamba-retry mambabuild conda/recipes/libcuml
+rapids-mamba-retry mambabuild \
+  --channel "${LIBCUDF_CHANNEL}" \
+  conda/recipes/libcuml
```

Comment:
`libcudf` is not a depencency of `libcuml` just for context. These chanes are just temporary to test things, right?

Reply:
Yup, all the `.sh` file changes will be reverted before merging this PR. Hence tagged it `DO NO MERGE` for now.